app/src/main/java/de/dreier/mytargets/features/statistics/CsvExporter.kt

```diff
@@ -114,11 +115,13 @@ class CsvExporter(private val context: Context, database: AppDatabase) {
             csv.add((e.index + 1).toString())
             // Timestamp
             csv.add(e.saveTime!!.format(DateTimeFormatter.ISO_LOCAL_TIME))
-            for ((_, index, _, x, y, scoringRing) in endDAO.loadShots(e.id)) {
+            for ((_, index, _, x, y, scoringRing, arrowNumber) in endDAO.loadShots(e.id)) {
                 csv.enterScope()
                 // Score
                 csv.add(target.zoneToString(scoringRing, index))
 
+                csv.add(if (arrowNumber == null) "" else arrowNumber)
```

Comment:
Please use the more compact `?:` instead

```
arrowNumber ?: ""
```

Reply:
The code was changed